qemu/tests/qemu_guest_agent.py

```diff
@@ -1458,6 +1458,63 @@ def gagent_check_frozen_io(self, test, params, env):
 
         self.gagent_verify(self.params, self.vm)
 
+    @error_context.context_aware
+    def gagent_check_vss_status(self, test, params, env):
+        """
+        Only for windows guest,check QEMU Guest Agent VSS Provider service start type
+        and if it works.
+
+        Steps:
+        1) Check VSS Provider service start type.
+        2) Check VSS Provider service should be in stopped status.
+        3) Issue fsfreeze qga command.
+        4) Check VSS Provider service should be in running status.
+        5) Issue fsthaw qga command.
+
+        :param test: kvm test object
+        :param params: Dictionary with the test parameters
+        :param env: Dictionary with test environment
+        """
+        def check_vss_status(status_expect):
+            error_context.context("Check VSS Provider status.", logging.info)
+            cmd_VSS_status = "sc query \"QEMU Guest Agent VSS Provider\" | findstr /i STATE"
+            status, output = session.cmd_status_output(cmd_VSS_status)
+            if status:
+                test.error("Command to check VSS service status failed,"
+                           "detailed info is:\n%s" % output)
+            VSS_status = output.split()[-1]
```

Comment:
could also change these variants to lowercase letters? 
VSS_status, VSS_start_type, cmd_VSS_start_type and the case name gagent_VSS_status

Reply:
@peixiu Updated all the uppercase letters.Could you have a review?
Thanks.